src/toast/timing.py

```diff
@@ -17,20 +20,55 @@
 from .utils import Environment
 
 
+stack_toast_file_pat = re.compile(r".*(toast.*)\.py")
+
+stack_toast_skip = [
```

Comment:
This list could do with some comments. Why where these functions singled out?

Reply:
I agree, it does seem arbitrary.  Basically these are cases where a base class method is just calling an internal derived class method and we don't want to clutter things.  However, I will think if there is some way to register these skipped names in the individual classes where they are defined.

Reply:
I have added a new decorator which adds a function / method to this global list of things to skip.  This is now used for the `Operator` and `Template` base class methods that are simply dispatch methods to call derived class code (so we don't have to see them in this simple trace).